react/ProductHighlights.tsx

```diff
@@ -122,6 +108,62 @@ const ProductHighlightContextProvider: FC<ProductHighlightContextProviderProps>
   )
 }
 
+ProductHighlights.propTypes = {
+  filter: PropTypes.object,
+  type: PropTypes.string
+}
+
+ProductHighlights.defaultProps = {
+  filter: {
+    type: 'hide',
+    highlightNames: [],
+  },
+  type: 'collection',
+}
```

Comment:
We don't really use `prop-types` anymore. It's present only on old components, now we rely solely on typescript. You can remove that and move the default values to the component declaration.

Reply:
@icazevedo 

I'm making the corrections you suggested, but the following error appears in the schema.

![code-structure](https://user-images.githubusercontent.com/80407814/120862078-b8070e80-c55e-11eb-8fba-d3d3dcbeb944.png)

Reply:
You can make ProductHighlights a function, like:

```tsx
function ProductHighlights({/* props here*/}: ProductHighlightsProps) {
}
```

And then it should work